### DISCUSSION TITLE
Difference between filament shrinkage and profile XY compensation

### DISCUSSION THREAD

**Discussion by tommy536** [OrcaSlicer/OrcaSlicer#541] [General]
In the new version 1.5.0 there is a new filament option shrinkage. 

What is the difference between the shrinkage and the profile settings "hole and contour compensation"?

**Comment by Brace91** [comment]
If you make a part with two 3mm holes, 100mm apart.

Shrinkage is if the two holes get closer together.  Ex: The center to center is now 99mm, set your shrinkage to 99%(being used to scaling the parts, I did this backwards the first try).

Hole compensation is if the individual holes get bigger or smaller.  Ex: The 3mm hole size is now 2.8mm, set the hole compensation to .1

After you've adjusted for shrinkage and the part is still big, you may need to adjust your contour compensation. 
Ex:  After adjusting shrinkage and your holes are back at 100mm Center to center the part is now .2mm too big. Adjust the contour compensation to -.1mm
